packages/js-moi-providers/src.ts/index.ts

```diff
@@ -2,7 +2,10 @@ export * from "./abstract-provider";
 export * from "./base-provider";
 export * from "./jsonrpc-provider";
 export * from "./voyage-provider";
-export * from "./websocket-provider";
-export type * from "../types/jsonrpc";
+// export * from "./websocket-provider";
```

Comment:
if not required remove it.

Reply:
https://github.com/sarvalabs/js-moi-sdk/pull/91/commits/099321f7ffa8fda363ecc7f082c979eb6ceb05c2